api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -356,70 +355,100 @@ public void getDateFormat_shouldReturnAPatternWithFourYCharactersInIt() {
 	 */
 	@Test
 	public void containsUpperAndLowerCase_shouldReturnTrueIfStringContainsUpperAndLowerCase() {
-		Assert.assertTrue(OpenmrsUtil.containsUpperAndLowerCase("Hello"));
-		Assert.assertTrue(OpenmrsUtil.containsUpperAndLowerCase("methodName"));
-		Assert.assertTrue(OpenmrsUtil.containsUpperAndLowerCase("the letter K"));
-		Assert.assertTrue(OpenmrsUtil.containsUpperAndLowerCase("The number 10"));
+		String str = "Hello";
```

Comment:
@sharif what was the intention behind modifying the existing tests ??

Reply:
@mozzy you are right but is that how assertions must be presented
i refactored that code because i think it didnot folllow testing convetions expecially on assertions
    it beared only actual but not expected code